src/gasket_page_table.c

```diff
@@ -56,6 +56,8 @@
 #include "gasket_constants.h"
 #include "gasket_core.h"
 
+MODULE_IMPORT_NS(DMA_BUF);
```

Comment:
This will need `#if kernel version >= ...` wrapping, or the build will break for older kernels.

I ran into that issue as well. I do have a CLA in place and will send an own PR, addressing this along that. Just that you are not surprised, @yan-wyb.